packages/now-build-utils/src/prerender.ts

```diff
@@ -7,21 +7,24 @@ interface PrerenderOptions {
   expiration: number;
   lambda: Lambda;
   fallback: FileBlob | FileFsRef | FileRef;
-  group: number;
+  group?: number;
 }
 
 export class Prerender {
   public type: 'Prerender';
   public expiration: number;
   public lambda: Lambda;
   public fallback: FileBlob | FileFsRef | FileRef;
-  public group: number;
+  public group?: number;
 
   constructor({ expiration, lambda, fallback, group }: PrerenderOptions) {
     this.type = 'Prerender';
     this.expiration = expiration;
     this.lambda = lambda;
     this.fallback = fallback;
-    this.group = group;
+    
+    if (group) {
```

Comment:
What's the purpose of using `if` here? Seems like it will be undefined either way 🤔

Reply:
Easier to read 😊

Reply:
I think less code is easier to read. Removing it will be logically the same and (arguably) easier to read.

Reply:
true, also could group be 0?